copy_encrypted_ami.sh

```diff
@@ -183,14 +187,31 @@ while read snapshotid; do
 done <<< "$SNAPSHOT_IDS"
 echo -e "${COLOR}EBS Snapshots copies completed ${NC}"
 
-# Prepares the json data with the new snapshot IDs and remove unecessary information
 sLen=${#SRC_SNAPSHOT[@]}
 
+# Prepares the json data with the new snapshot IDs and remove unecessary information
```

Comment:
Any reason why you moved the comment to after the sLen assignment ?

Reply:
Not really. 
I was going to do some stuffs but abandonned. Then I forgot to move it back. Fixed in latest commit.